src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -575,7 +575,16 @@ public static void AddCertificateChainToTheUserCertificateAuthorityStoreAndClean
                     {
                         if (element.Certificate.NotAfter.ToUniversalTime() > utcNow && element.Certificate.NotBefore.ToUniversalTime() < utcNow)
                             continue;
-                        userIntermediateStore.Remove(element.Certificate);                        
+                        try
+                        {
+                            userIntermediateStore.Remove(element.Certificate);
```

Comment:
@iftahbe the loop can continue after catching the exception? Can we succeed on some of the certificates and fail on others or we will just log a lot of exceptions?

Reply:
Yes we can fail on one and succeed at the other. I believe this is a very rare case. Anyway, I'm moving the logging to Info.